cmd/apiserver_receive_adapter/main.go

```diff
@@ -107,7 +147,10 @@ func main() {
 			logger.Fatalw("Error parsing APIVersion", zap.Error(err))
 		}
 		// TODO: pass down the resource and the kind so we do not have to guess.
-		gvr, _ := meta.UnsafeGuessKindToResource(schema.GroupVersionKind{Kind: kind, Group: gv.Group, Version: gv.Version})
+		gvr, _ := meta.UnsafeGuessKindToResource(schema.GroupVersionKind{
```

Comment:
again here... so that we minimize the changes in the PR...

Reply:
sure, but that's hard to read IMO.

Reply:
fixed